tests/sentry/tsdb/test_snuba.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import absolute_import
+
+import pytz
+from datetime import datetime, timedelta
+
+from sentry.testutils.cases import OutcomesSnubaTest
+from sentry.tsdb.base import TSDBModel
+from sentry.tsdb.snuba import SnubaTSDB
+from sentry.utils.outcomes import Outcome
+
+
+def to_epoch_time(value):
+    value = value.replace(minute=0, second=0, microsecond=0)
+    return int((value - datetime(1970, 1, 1, tzinfo=pytz.utc)).total_seconds())
+
+
+class SnubaTSDBTest(OutcomesSnubaTest):
+    def setUp(self):
+        super(SnubaTSDBTest, self).setUp()
+        self.db = SnubaTSDB()
+
+        # Set up the times
+        self.now = datetime.now(pytz.utc)
+        self.skew_days = 7
+        self.skew = timedelta(days=self.skew_days)
+        self.start_time = self.now - self.skew
+        self.one_day_later = self.start_time + timedelta(days=1)
```

Comment:
We have `sentry.testutils.helpers.datetime.before_now()` to help generate datetimes that are offset from utcnow. Might help here.

Reply:
Tried doing this but ran into a timezone issue. I think its because tsdb/base.py uses [`to_timestamp`](https://github.com/getsentry/sentry/blob/master/src/sentry/tsdb/base.py#L189), which has it's [epoch defined with a pytz.utc](https://github.com/getsentry/sentry/blob/master/src/sentry/utils/dates.py#L14) and `before_now` uses `utcnow()`, which doesn't attach a timezone.

Also, I need `self.now` so think might not be a bad idea to just do the calculation in the test.